src/components/organisms/AdminVenueView/components/EventCard/EventCard.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import classNames from "classnames";
+import { sortBy } from "lodash";
+
+import { VenueEvent } from "types/venues";
+
+import { isEventLive } from "utils/event";
+import { WithId, WithVenueId } from "utils/id";
+import { formatTimeLocalised } from "utils/time";
+import { adminNGVenueUrl } from "utils/url";
+
+import { useVenueId } from "hooks/useVenueId";
+
+import { AdminVenueTab } from "components/organisms/AdminVenueView/AdminVenueView";
+
+import "./EventCard.scss";
+
+interface EventCardProps {
+  events?: WithVenueId<WithId<VenueEvent>>[];
+}
+
+export const EventCard: React.FC<EventCardProps> = ({ events }) => {
+  const venueId = useVenueId();
+  const [nextEvent, followingEvent] = sortBy(events ?? [], "start_utc_seconds");
+  const live = nextEvent && isEventLive(nextEvent);
+
+  const parentClasses = classNames({
+    "EventCard EventCard--no-event": !nextEvent,
+    "EventCard EventCard--live-event": nextEvent && live,
+    "EventCard EventCard--future-event": nextEvent && !live,
+  });
```

Comment:
```suggestion
  const parentClasses = classNames("EventCard", {
    "EventCard--no-event": !nextEvent,
    "EventCard--live-event": nextEvent && live,
    "EventCard--future-event": nextEvent && !live,
  });
```

Reply:
@sunny-viktoryia as a rule of thumb `EventCard` at the parent level divorced from any kind of modifier helps outside targeting of elements. Example
```tsx
<ComponentWithChildren><OneChild /></ComponentWithChildren>
```
would allow for current SCSS to target the children inside it by simply using
```scss
.ComponentWithChildren .OneChild { color: $primary }
```
without the need of guessing what is the current modifier on the parent component